### PR TITLE
Modularised Deploying Booster in REST API mission

### PR DESCRIPTION
**Pull request by tradej**
Resolves [OBST-420](https://issues.jboss.org/browse/OBST-420).

Rendered docs are in my private space (the links in docs do not work because they are relative):
* [Eclipse Vert.X Guide](http://download.eng.brq.redhat.com/scratch/tradej/rhoar/OBST-420/html/vertx-runtime.html)
* [Node.js Guide](http://download.eng.brq.redhat.com/scratch/tradej/rhoar/OBST-420/html/nodejs-runtime.html)
* [Spring Boot Guide](http://download.eng.brq.redhat.com/scratch/tradej/rhoar/OBST-420/html/spring-boot-runtime.html)
* [WF Swarm Guide](http://download.eng.brq.redhat.com/scratch/tradej/rhoar/OBST-420/html/wf-swarm-runtime.html)

### CODE REVIEW

File: docs/topics/mission-rest-api/assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+[#building-and-deploying-the-rest-api-booster-to-openshiftlocal]
+= Building and Deploying the {name-mission-http-api} Booster to {OpenShiftLocal}
+
+You have two options how to execute the {name-mission-http-api} booster locally on {OpenShiftLocal}:
```
Comment (rhoads-zach) [inline docs/topics/mission-rest-api/assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc]: Consider rephrasing to:

You have two options for to executing the {name-mission-http-api} booster locally on {OpenShiftLocal}

File: docs/topics/mission-rest-api/assembly_building-and-deploying-the-booster-to-openshiftonline.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+[#building-and-deploying-the-rest-api-booster-to-openshiftonline]
+= Building and Deploying the {name-mission-http-api} Booster to {OpenShiftOnline}
+:context: openshiftonline
+
+You have two options how to execute the {name-mission-http-api} booster on {OpenShiftOnline}:
```
Comment (rhoads-zach) [inline docs/topics/mission-rest-api/assembly_building-and-deploying-the-booster-to-openshiftonline.adoc]: same as the previous

File: docs/topics/proc_getting-launcher-url-and-credentials-on-openshiftlocal.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+You need the {launcher} application URL and user credentials to create and deploy boosters on {OpenShiftLocal}. These data are provided by the {OpenShiftLocal} binary when you start it.
+
+.Prerequisites
+
+* The {launcher} application installed, configured, and running. For more information, see the link:{link-launcher-openshift-local-install-guide}[{openshift-local-guide-name}].
+
+.Procedure
+
+. Navigate to the console where you started {OpenShiftLocal}
```
Comment (rhoads-zach) [inline docs/topics/proc_getting-launcher-url-and-credentials-on-openshiftlocal.adoc]: Period at the end of this sentence.

**Comment by tradej** [comment]
Thanks for the preliminary review, @rhoads-zach. The change is now ready. I have moved the assemblies back with other files, so now there shouldn't be any issues.

**Comment by rhoads-zach** [comment]
LGTM, merging.

**Comment by rhoads-zach** [comment]
merged: d1d3c6520c6d118e2d3aba857fb1db0a9426f89a

**Pull request closed without merging**